web-app/src/app/tabs/faults/faults.component.html

```diff
@@ -1,5 +1,130 @@
+		<div class="item off" [ngClass]="{'on' : motor1Faults.errorFlags.configReadError == true, 'off' : motor1Faults.errorFlags.configReadError == false}">Config Read Error</div>
+		<div class="item off" [ngClass]="{'on' : motor1Faults.errorFlags.rail15VUnderVoltageLockOut == true, 'off' : motor1Faults.errorFlags.rail15VUnderVoltageLockOut == false}">Rail Under Voltage Lock Out</div>
+		<div class="item off" [ngClass]="{'on' : motor1Faults.errorFlags.desaturationFault == true, 'off' : motor1Faults.errorFlags.desaturationFault == false}">Desaturation Fault</div>
+	</div>
+	<div class="rightside">
+		<div class="subheader">Motor 1 Limits</div>
+		<div class="item off" [ngClass]="{'on' : motor1Faults.limitFlags.outputVoltagePwm == true, 'off' : motor1Faults.limitFlags.outputVoltagePwm == false}">Output Voltage PWM</div>
+		<div class="item off" [ngClass]="{'on' : motor1Faults.limitFlags.motorCurrent == true, 'off' : motor1Faults.limitFlags.motorCurrent == false}">Motor Current</div>
+		<div class="item off" [ngClass]="{'on' : motor1Faults.limitFlags.velocity == true, 'off' : motor1Faults.limitFlags.velocity == false}">Velocity</div>
+		<div class="item off" [ngClass]="{'on' : motor1Faults.limitFlags.busCurrent == true, 'off' : motor1Faults.limitFlags.busCurrent == false}">Bus Current</div>
+		<div class="item off" [ngClass]="{'on' : motor1Faults.limitFlags.busVoltageUpper == true, 'off' : motor1Faults.limitFlags.busVoltageUpper == false}">Bus Voltage Upper</div>
+		<div class="item off" [ngClass]="{'on' : motor1Faults.limitFlags.busVoltageLower == true, 'off' : motor1Faults.limitFlags.busVoltageLower == false}">Bus Voltage Lower</div>
+		<div class="item off" [ngClass]="{'on' : motor1Faults.limitFlags.ipmOrMotorTemperature == true, 'off' : motor1Faults.limitFlags.ipmOrMotorTemperature == false}">IPM Or Motor Temperature</div>
+	</div>
+</div>
+<div class="container-row full">
+	<div class="leftside">
+		<div class="subheader">RX Error Count</div>
+		<div class="container-row">
+			<div class="leftside">
+				<div class="data item"> Motor 0 </div>
+				<div class="data item value">{{motor0Faults.rxErrorCount}}</div>
+			</div>
+			<div class="rightside">
+				<div class="data item"> Motor 1 </div>
+				<div class="data item value">{{motor1Faults.rxErrorCount}}</div>
+			</div>
+		</div>
+	</div>
+	<div class="rightside">
+		<div class="subheader">TX Error Count</div>
+		<div class="container-row">
+			<div class="leftside">
+				<div class="data item"> Motor 0 </div>
+				<div class="data item value">{{motor0Faults.txErrorCount}}</div>
+			</div>
+			<div class="rightside">
+				<div class="data item"> Motor 1 </div>
+				<div class="data item value">{{motor1Faults.txErrorCount}}</div>
+			</div>
+		</div>
+	</div>
+</div>
+<!-- 	<div>
+		<div>RX Error Count</div>
+		<div>TX Error Count</div>
+	</div> -->
+<!-- 	<div>
+		<div>RX Error Count</div>
+		<div>TX Error Count</div>
+	</div> -->
+<!-- 	<div class="container-row">
```

Comment:
I'm guessing this is just for future work?

Reply:
Nope, I just completely forgot to remove this LOL